airflow-ctl/docs/installation/installing-from-pypi.rst

```diff
@@ -68,15 +68,15 @@ Constraints files
 Why we need constraints
 =======================
 
-Airflow CTL installation can be tricky because Airflow CTL is both a library and an application.
+airflowctl installation can be tricky because airflowctl is both a library and an application.
```

Comment:
I think that whole chapter can be easily removed. I do not see airflowctl having the same issue as `airflow-core` -> we can easily treat it as an application and we do not have to use constraints for it.

Reply:
No, that's too much :D I think I overlooked this 😅 Let me run this with profiler and give a rough estimate on this :) I think it should be able to run even in one of the smallest Docker containers that has Python :D